docs/mdfiles/simulcast.md

```diff
@@ -0,0 +1,71 @@
+OWT Simulcast Description
+---------------------
+
+# {#simulcast}
+The MCU server supports simulcast. This can be enabled through OWT Client SDK API. <br>
```

Comment:
MCU -> conference. It also supports forwarding mode.

`<br>` is not required.

Reply:
This can be enabled through OWT Client SDK for JavaScript. Only JavaScript SDK can enable simulcast.